python/ql/test/library-tests/frameworks/aiohttp/InlineTaintTest.ql

```diff
@@ -1 +1,2 @@
 import experimental.meta.InlineTaintTest
+import MakeInlineTaintTest<TestTaintTrackingConfig>
```

Comment:
I wonder if this could be put into a file, say `DefaultInlineTaintTest.qll` that is then imported. Just since this is the way we always use it...

Reply:
I generally did that when the config and the `Make....` were in the same file and when it was clear to me that it would not be sensible to use any other configuration. If you think that the latter applies here (i.e., no other sensible configuration), then I happily move this.

Reply:
It might be sensible we just do not really do it. That is why I suggested a convenience file so a single import would be enough in most cases. It could even be named `InlineTaintTest` to remove the common case form the diff of this PR; then we just have to come up with a different name for the unspecialised one...

Reply:
I do not think such an ergonomic improvement _has_ to be part of this PR, though. The common workflow is to just copy an existing version of the file into a new test folder, so the number of lines in the file does not really matter inthat case...

Reply:
Then, I'll leave that improvement to the Python team.